server/src/test/proto/spine/test/clientservice/customer/customer.proto

```diff
@@ -46,4 +46,7 @@ message Customer {
 
     // The name of the customer.
     people.PersonName name = 2;
+
+    // Info about customer's family
+    repeated people.PersonName family = 3;
```

Comment:
`family` cannot be a set of names. The naming is wrong.
Also you are missing a `.` symbol.